### PR TITLE
super-productivity: 9.0.5 -> 10.0.2

### PR DESCRIPTION
**Pull request by Masrkai**
## Description of changes

Github release is 10.0.2 as of 7:26 9/9/2024 (GMT+3)
https://github.com/johannesjo/super-productivity/releases/tag/v10.0.2

Comparison:
https://github.com/johannesjo/super-productivity/compare/v9.0.5...v10.0.2

Home Page:
https://super-productivity.com/

## Things done
Updated the package version and hash and replaced the older sha256

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/office/super-productivity/default.nix [hunk 1]
```diff
@@ -2,16 +2,17 @@
...
     url = "https://github.com/johannesjo/super-productivity/releases/download/v${version}/superProductivity-${version}.AppImage";
-    sha256 = "sha256-eNAoLcQWnsTDA7sG8i0Ur9BZ+pNt4AK1GOppFCD1ZGg=";
+    hash = "sha256-TPe2wijTzlro9rtESJHPL1phtraSW5kTHfctSTTumGs=";
     name = "${pname}-${version}.AppImage";
   };
 
   appimageContents = appimageTools.extractType2 {
-    inherit pname version src;
+    name = "${pname}-${version}";
+    inherit src;
```
Comment (eclairevoyant) [inline pkgs/applications/office/super-productivity/default.nix:15]: Why? Please remove this change.

File: pkgs/applications/office/super-productivity/default.nix [hunk 2]
```diff
@@ -41,11 +42,12 @@ stdenv.mkDerivation rec {
...
+    description = "To-do list & time tracker for programmers and other digital workers with Jira, Github, and Gitlab integration";
     homepage = "https://super-productivity.com";
+    downloadPage = "https://github.com/johannesjo/super-productivity";
     license = licenses.mit;
     platforms = [ "x86_64-linux" ];
     maintainers = with maintainers; [ offline ];
     mainProgram = "super-productivity";
   };
-}
+}
```
Comment (eclairevoyant) [inline pkgs/applications/office/super-productivity/default.nix:53]: This change isn't needed either (and is flagged by CI)

**Review by eclairevoyant — CHANGES_REQUESTED** [review]

File: pkgs/applications/office/super-productivity/default.nix (hunk 1 above)
Comment (Masrkai) [reply pkgs/applications/office/super-productivity/default.nix:15]: okay, thanks for the help it's the first time i ever contribute to nixpkgs

**Comment by eclairevoyant** [comment]
Why close it? The rest of the PR is probably fine

**Comment by Masrkai** [comment]
> Why close it? The rest of the PR is probably fine

becasue i can't figure out how to commit to this PullRequest it's an issue on my part so i am sorry

**Comment by Masrkai** [comment]
i made a PR with the exactly the needed changes just now so check it instead

**Pull request closed without merging**